_data/security.yml

```diff
@@ -89,15 +89,6 @@ websites:
       tfa: No
       twitter: gogetssl
 
-    - name: Imperva Incapsula
-      url: https://www.incapsula.com/
-      img: incapsula.png
-      tfa: Yes
-      email: Yes
-      sms: Yes
-      software: Yes
-      doc: https://docs.incapsula.com/Content/Management%20Console%20and%20Settings/accountsettings.htm
```

Comment:
Why was Incapsula removed? They seem to still provide service, and they have a doc page here: `https://docs.incapsula.com/Content/management-console-and-settings/account-settings.htm`